api/controllers/WebActionsController.js

```diff
@@ -238,11 +238,14 @@ module.exports = {
         // If available, attach the referralCode to the redirect URL
         if (referralCode.length > 0) {
           redirectUrl += `&referralCode=${referralCode}`;
-
+          // Get partner/campaign name to identify where a user signed up
+          let partnerOrCampaign = req.body.partner
```

Comment:
When both are true it should use the first. Either way, the ordering doesn't matter too much in this case since there shouldn't be a case right now where both could be true.